config/locales/en/course/mailer/codaveri_feedback_email.yml

```diff
@@ -0,0 +1,11 @@
+en:
+  course:
+    mailer:
+      codaveri_feedback_email:
+        subject: '%{course} - New Codaveri feedback on %{topic}'
+        recipients: 'Course Managers'
+        message: >
+          <p>Automated code annotations have been generated by Codaveri and are now ready for your review.</p>
```

Comment:
<p>[Beta] Automated code annotations have been generated by Codaveri for your review.</p><p>Please help to update the comment to what you think is the most instructive/helpful to the student's learning, or delete the comment if you feel no comment is required.</p><p>Your comments will be used to improve future code annotations generated by Codaveri.</p>

Reply:
Thanks!